octobot_trading/exchange_data/ohlcv/candles_manager.py

```diff
@@ -98,6 +104,24 @@ def get_symbol_prices(self, limit=-1):
             enums.PriceIndexes.IND_PRICE_TIME.value: self.get_symbol_time_candles(limit)
         }
 
+    def get_candles(self, limit=-1):
```

Comment:
Where is it used?

Reply:
in `AbstractScriptedTradingMode#_wait_for_symbol_init` to call candles callback for the init call when enough data have been fetched

Reply:
ok